### PR TITLE
New Session and route

### PR DESCRIPTION
**Pull request by s-zimm**
Links handlebar template with Postgres database for new session. Generates games in form field for users to choose. Skeleton for new session submission form at route /session/new. Also adds new games for test data.

### CODE REVIEW

**Comment by s-zimm** [comment]
@Vurolock

File: views/session-form.hbs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<div class="container">
+    <form action="/scores" method="POST">
+        <div class="form-group">
+            <label for="game">Choose your game:</label>
+                <select multiple name="game" id="game" class="form-control" size="8">
+                    {{#each games}}
+                    <option value={{id}}>{{name}}</option>
+                    {{/each}}
+                    {{!-- <option value="">s</option>
+                    <option value="">sd</option>
```
Comment (wegmans-andrew-keller) [inline views/session-form.hbs]: lets take out these test lines before we merge

Comment (s-zimm) [reply views/session-form.hbs]: Good call - will do.

**Review by wegmans-andrew-keller — COMMENTED** [review]
Looking good

**Pull request merged**